host-configs/LLNL/lassen-xl@2020.08.13.cmake

```diff
@@ -1,56 +0,0 @@
-set(CONFIG_NAME "lassen-xl@2020.08.13" CACHE PATH "")
```

Comment:
what happened to this?

Reply:
It didn't have an associated set of TPLs, unlike the other LC configs. If you want to build with XL the easiest thing to do is to just use spack to build the dependencies.